clouds/bigquery/modules/doc/h3/H3_POLYFILL.md

```diff
@@ -6,9 +6,9 @@ carto.H3_POLYFILL(geography, resolution)
 
 **Description**
 
-Returns an array with all the H3 cell indexes **with centers** contained in a given polygon. It will return `null` on error (invalid geography type or resolution out of bounds).
+Returns an array with all the H3 cell indexes **with centers** contained in a given polygon. It will return `null` on error (invalid geography type or resolution out of bounds). In case of lines, it will return the H3 cell indexes intersecting those lines. For a given point, it will return the H3 index of cell in which that point is contained.
```

Comment:
I think we should add a comment regarding the known issues, that in some cases some cells might be missing due to the approximations we do